api/transactionHandler/metamorph.go

```diff
@@ -29,7 +29,21 @@ type Metamorph struct {
 }
 
 // NewMetamorph creates a connection to a list of metamorph servers via gRPC
-func NewMetamorph(targets string, blockTxClient blocktx.ClientI, grpcMessageSize int, logger utils.Logger) (*Metamorph, error) {
+func NewMetamorph(address string, blockTxClient blocktx.ClientI, grpcMessageSize int, logger utils.Logger) (*Metamorph, error) {
+	conn, err := GetConnection(address, grpcMessageSize)
+	if err != nil {
+		return nil, err
```

Comment:
maybe wrap this into additional message like "Unable to get connection due to ", err or something like that ? )